package.json

```diff
@@ -160,24 +160,24 @@
     "cozy-authentication": "2.1.0",
     "cozy-bar": "^8.11.2",
     "cozy-ci": "0.4.1",
-    "cozy-client": "33.2.0",
+    "cozy-client": "^34.5.0",
     "cozy-client-js": "0.16.4",
-    "cozy-device-helper": "^2.1.0",
+    "cozy-device-helper": "^2.6.0",
     "cozy-doctypes": "1.82.2",
     "cozy-flags": "^2.8.7",
-    "cozy-harvest-lib": "^9.32.4",
+    "cozy-harvest-lib": "^10.0.0",
```

Comment:
Is it too early to update to 11.0.0?

Reply:
on pourrait, mais il faut également faire monter d'autre package... à faire dans une prochaine PR